plugins/cameraserver/src/main/java/edu/wpi/first/shuffleboard/plugin/cameraserver/CameraServerPlugin.java

```diff
@@ -0,0 +1,69 @@
+package edu.wpi.first.shuffleboard.plugin.cameraserver;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+
+import edu.wpi.cscore.CameraServerJNI;
+import edu.wpi.first.shuffleboard.api.data.DataType;
+import edu.wpi.first.shuffleboard.api.plugin.Plugin;
+import edu.wpi.first.shuffleboard.api.sources.SourceType;
+import edu.wpi.first.shuffleboard.api.widget.Widget;
+import edu.wpi.first.shuffleboard.plugin.cameraserver.data.type.CameraServerDataType;
+import edu.wpi.first.shuffleboard.plugin.cameraserver.source.CameraServerSourceType;
+import edu.wpi.first.shuffleboard.plugin.cameraserver.widget.CameraServerWidget;
+
+import org.opencv.core.Core;
+
+import java.util.List;
+import java.util.Map;
+import java.util.logging.Logger;
+
+public class CameraServerPlugin extends Plugin {
+
+  private static final Logger log = Logger.getLogger(CameraServerPlugin.class.getName());
+
+  @SuppressWarnings("JavadocMethod")
+  public CameraServerPlugin() {
+    super("edu.wpi.first.shuffleboard",
+        "CameraServer",
+        "1.0.0",
+        "Provides sources and widgets for viewing CameraServer MJPEG streams.");
+  }
+
+  @Override
+  public void onLoad() {
+    log.info("OpenCV version: " + Core.VERSION);
+    // Make sure the JNI is loaded. If it's not, this plugin can't work!
+    // Calling a function from CameraServerJNI will extract the OpenCV JNI dependencies and load them
+    CameraServerJNI.getHostname();
```

Comment:
I think this is fine to merge as-is, but we should probably file a CameraServer bug about the API gap here.